hs/Database/CourseQueries.hs

```diff
@@ -127,10 +128,19 @@ buildSession lecs tuts =
 
 -- ** Other queries
 
+-- | Gets Graph Id using graph title
+getGraphId :: String -> IO Int64
```

Comment:
I don't think `getGraphJSON` is used anywhere else; if it isn't, you can change it directly to take in a string rather than Int64.